hydradx/model/amm/centralized_market.py

```diff
@@ -0,0 +1,174 @@
+import copy
+from .agents import Agent
+from .amm import AMM
+
+
+class OrderBook:
```

Comment:
I would suggest adding logic to this which sorts the orders on instantiation.

Also add the ability to add/remove/modify (i.e. partially fill) orders while keeping it sorted.

Maybe also a function which checks if the order book has self-arbitrage (i.e. if the lowest ask is <= the highest bid).